docs/includes/apiargs-MongoDBCollection-method-createIndex-option.yaml

```diff
@@ -72,9 +72,8 @@ description: |
   Specifies the :manual:`version of a 2dsphere </core/2dsphere>` index to
   create.
 
-  MongoDB 2.6 introduced version 2 of 2dsphere indexes. Version 2 is the default
-  version of 2dsphere indexes created in MongoDB 2.6 and later versions.
-  ``2dsphereIndexVersion`` enables you to override the default version 2.
+  Version 2 is the default version of 2dsphere indexes. ``2dsphereIndexVersion``
+  enables you to override the default version 2.
```

Comment:
It looks like MongoDB 3.2 changed the default version to 3, so this is quite out of date :)

I think we can just remove this paragraph entirely and keep a single sentence above. The current language might encourage users to supply this option, so I'd suggest we change it to:

```
Overrides the server's default version for  a :manual:`2dsphere </core/2dsphere>` index.
```

I noticed the MongoDB manual was also outdated and reported that in [DOCS-11278](https://jira.mongodb.org/browse/DOCS-11278).